src/core/mod.rs

```diff
@@ -6,6 +6,10 @@ pub use oauth2::basic::{
     BasicRequestTokenError as CoreRequestTokenError,
     BasicRevocationErrorResponse as CoreRevocationErrorResponse, BasicTokenType as CoreTokenType,
 };
+
+use oauth2::devicecode::{DeviceAuthorizationResponse, EmptyExtraDeviceAuthorizationFields};
+pub use oauth2::DeviceAuthorizationUrl as CoreDeviceAuthorizationUrl;
```

Comment:
I think we can remove this since it's already being exported from the top-level `lib.rs` without the `Core` prefix and this isn't a trait or generic type that needs a typedef